code/modules/mob/living/emote.dm

```diff
@@ -34,6 +34,12 @@
 	message = "crosses their arms."
 	restraint_check = TRUE
 
+/datum/emote/living/crack
+	key = "crack"
+	key_third_person = "cracks"
+	message = "cracks their knuckles together."

```

Comment:
what if you only have one hand

this should be "cracks their knuckles." instead

Reply:
Fixed it with your suggestion.